sift.gemspec

```diff
@@ -19,12 +19,13 @@ Gem::Specification.new do |s|
                 "Rakefile",
                 "README.md"]
 
-  s.required_ruby_version = ">= 2.3.0"
+  s.required_ruby_version = ">= 2.7.0"
 
-  s.add_dependency "rails", "> 4.2.0"
+  s.add_dependency "activerecord", ">= 6.1"
+  s.add_dependency "net-http"
 
   s.add_development_dependency "pry"
-  s.add_development_dependency "rails", ">= 5.1"
+  s.add_development_dependency "rails", ">= 7.0"
```

Comment:
NB: I guess it will be better to align these two versions:
```
s.add_dependency "activerecord", ">= 6.1"
s.add_development_dependency "rails", ">= 7.0"
```